src/main/java/io/jenkins/plugins/sprp/PipelineSnippetGenerator.java

```diff
@@ -291,12 +291,12 @@ else if(stepFieldClass == Integer.class)
                 if (stage.getName().equals("Tests")) {
                     if(testResultPaths != null)
                         snippetLines.addAll(getPublishReportSnippet(testResultPaths));
-//                    TODO Abhishek: code is commented out for testing purposes, it will be reinstated later
-//                    if(gitConfig.getGitUrl() != null)
-//                        snippetLines.add("gitPush " +
-//                                "credentialId: \"" + gitConfig.getCredentialsId() + "\"," +
-//                                "url: \"" + gitConfig.getGitUrl() + "\"," +
-//                                "branch: \"" + gitConfig.getGitBranch() + "\"" );
+
+                    if(gitConfig.getGitUrl() != null)
+                        snippetLines.add("gitPush " +
+                                "credentialId: \"" + gitConfig.getCredentialsId() + "\"," +
+                                "url: \"" + gitConfig.getGitUrl() + "\"," +
+                                "branch: \"" + gitConfig.getGitBranch() + "\"" );
                 }
```

Comment:
The code above is confusing because you do not use brackets. The bracket on line 300 closes the bracket from line 291 (I think), but that is not obvious. This is why 1TBS is so important.